src/Classes/Models/HiringPartnerContactsModel.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Portal\Models;
+
+use Portal\Entity\HiringPartnerContactEntity;
+
+class HiringPartnerContactsModel
+{
+
+    private $db;
+
+    public function __construct(\PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    /**
+     * Queries the database to get an array of hiring partner company ids
+     *
+     * @return array
+     */
+    public function getHiringPartnerContactById() :array {
```

Comment:
Is this method supposed to get something by id? There is not input and in the statement, and you take everything from the database, not a specific one.